src/screens/DiscoveryScreenV2/fragment/NewsFragment.js

```diff
@@ -1,161 +1,140 @@
-                score: item.domain.credderScore,
-                follower: 0
-              }
-            });
-          };
-
-          // Disable on press content if view should be navigated to LinkContextScreen
-          if (!item.dummy)
-            return (
-              <RenderItem
-                key={`news-screen-item-${index}`}
-                item={item}
-                selfUserId={myId}
+    const [myId, setMyId] = React.useState('')
+    const [defaultNews] = React.useContext(Context).news
+
+    const isReady = useIsReady()
+
+    const navigation = useNavigation()
+
+    React.useEffect(() => {
+        const parseToken = async () => {
+            const id = await getUserId();
+            if (id) {
+                setMyId(id);
+            }
+        };
+        parseToken();
+    }, []);
+
+    // React.useEffect(() => {
+    //     if(news.length > 0) setIsFirstTimeOpen(false)
+    // }, [news])
+
+    const renderNewsItem = () => {
+        if (isFirstTimeOpen) {
+            return [<DiscoveryTitleSeparator key="news-title-separator" text="Suggested News" />].concat(defaultNews.news.map((item, index) => {
+                const onContentClicked = () => {
+                    navigation.navigate('DetailDomainScreen', {
+                        item: {
+                            id: item.id,
+                            score: item.domain.credderScore,
+                            follower: 0,
+                        }
+                    })
+                }
+
+                // Disable on press content if view should be navigated to LinkContextScreen
+                if (!item.dummy) return <RenderItem key={`news-screen-item-${index}`}
+                    item={item}
+                    selfUserId={myId}
                 // onPressContent={onContentClicked}
-              />
-            );
+                />
+            }))
+        }
+
+        return news.map((item, index) => {
+            const contentParam = newsDiscoveryContentParamBuilder(
+                item.title,
+                item.image,
+                item.description,
+                item.news_url,
+                item?.newsLinkDomain?.logo,
+                item?.newsLinkDomain?.domain_name || item?.site_name,
+                item.createdAt,
+                item.news_link_id,
+                item?.newsLinkDomain?.credder_score,
+                item?.post_id
+            )
+
+            const onContentClicked = () => {
+                navigation.navigate('DetailDomainScreen', {
+                    item: {
+                        id: item.post_id,
+                        score: item?.newsLinkDomain?.credder_score
+                    }
+                })
+            }
+
+            return <RenderItem key={`newsDiscovery-${index}`}
+                selfUserId={myId}
+                item={contentParam}
+                onPressShare={share.shareNews}
+            // onPressContent={onContentClicked} 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPressContent` prop is commented out in both instances of the `RenderItem` component. If this is intentional and you want to disable this functionality, it's fine. However, if this is an oversight, uncomment these lines to restore the functionality.
```diff
- // onPressContent={onContentClicked}
+ onPressContent={onContentClicked}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->